Dockerfile

```diff
@@ -1,19 +1,19 @@
 # [Node Stage to get node_modolues and node dependencies]
-FROM node:8.16.0-buster-slim as node_stage
+FROM node:8.16.0-buster-slim as node_base
+# [Python Stage for Django web server]
+FROM python:3.10.14-slim-bullseye as python_base
 
+FROM node_base as node_dependencies
 COPY ./yarn.lock yarn.lock
 COPY ./package.json package.json
 
 RUN apt-get update
 RUN apt-get install python-pip -y
 
 RUN npm install -g yarn
-RUN yarn install --dev --frozen-lockfile  \
```

Comment:
Do you mean move `yes | poetry cache clear --all pypi` in node_dependencies stage?

Reply:
no, i mean yarn's cache. you've deleted `rm -rf $HOME/.cache/yarn/*` and i wonder why?

Reply:
I didn't notice I deleted it. But do we still need it since it doesn't affect the final image (prod&dev)?

Reply:
> do we still need it since it doesn't affect the final image

Yes, you're correct, it does not affect the final image. 
I'd prefer to run the cleanup still just for caution. Someone could still mess up in the future and make the artifacts be carried over to the resulting image.

> might be yarn clean cache

yes, you're right. Yarn does not have this command when the repo is launched, so it simply runs the `rm`.